src/bioregistry/schema/schema.json

```diff
@@ -260,6 +260,11 @@
           "integration_status": "required_for_new",
           "type": "string"
         },
+        "rdf_uri_format": {
+          "title": "RDF URI format string",
+          "description": "The RDF URI format string, which must have at least one ``$1`` in it",
+          "type": "string"
```

Comment:
Does this allow one provider to provide more than one URI pattern/format?

Reply:
But I can also imagine that each URI format is a separate provider. Example: WikiPathways has WPRDF and GPMLRDF and each pathway has a IRI in each one of them, but with a different pattern.

Reply:
On the first pass, this doesn't allow this. I had considered adding a flag into the "providers" field, where additional ones can be annotated, but I didn't think adding this kind of complexity on the first pass. There's also an argument to be made that the RDF IRIs should be globally unique, but I will let people more invested argue that out ;) 

Can you please give an explicit example for wikipathways that we can come back to later?

Reply:
Yes, will look up two matching IRIs after lunch.

Reply:
- WPRDF: <https://identifiers.org/wikipathways/WP4860>
- GPMLRDF: <http://rdf.wikipathways.org/Pathway/WP4846_r124663>

The latter actually has the revision in the IRI, so that complicates things a lot. So, more than happy to just take the WPRDF URI pattern for now. The context is that the GPMLRDF is the RDF that describes the pathway model (the GPML as RDF), while the WPRDF is the biological process. See https://doi.org/10.1371/journal.pcbi.1004989 for exact details

Reply:
I don't think this is a very good example to make the case for a given semantic space having multiple RDF URIs - the inclusion of the revision makes it part of a different (albeit, related) semantic space. 

Here's a query over Wikidata for any property that has more than one RDF URI format, it doesn't appear to return any results: https://w.wiki/6j4n

Maybe you have a different example in mind? Like an example where UniProt/SIB has PURLs for something that also has a first-party PURL? @JervenBolleman maybe you know of an instance where this happened

Reply:
Well the worst case that I can think of is pubmed.  They when using the facebook/open graph protocol, state that the IRI of the pubmed:1 concept is `https://pubmed.ncbi.nlm.nih.gov/1/` in the RDF based OGP.

But there is also `http://rdf.ncbi.nlm.nih/pubmed/1` and there is of course also `http://rdf.ncbi.nlm.nih.gov/pubchem/reference/1`. 

This next to `http://purl.uniprot.org/citation/1`. n2t and identifiers.org. 

Note: the identifier in `http://purl.uniprot.org/citation/X` may be something else than a pubmed identifier.

Note: I once hope to hear from the Pubmed technical staff about the semantics of these different IRIs.

Note: NCBI Taxonomy is similar in it's problem space, but their tech stack is even older and budget to fix anything is minimal.

Note: For UniProt we try our hardest to not use fake uniprot purls for databases that have an RDF representation or thought about the semantics of their IRIs. When they exists, and we don't use them it is most likely because we don't know about them.

Reply:
> Well the worst case that I can think of is pubmed. They when using the facebook/open graph protocol, state that the IRI of the pubmed:1 concept is `https://pubmed.ncbi.nlm.nih.gov/1/` in the RDF based OGP.

Yes, another nice example. We have plenty of this issue in Open PHACTS, also for UniProt ;)

But I think we can argue that every pattern is provided by a different provider and "solve" it that way. Not?

Reply:
@egonw depends on what you want to annotate. e.g. provider recommended or just in use?

Reply:
I think Bioregistry is all models around providers, so I would say "provider recommended".

Unless you mean one provider recommends one thing and then does another. But that situation is perhaps similar to the above WikiPathways example where this provider (us) use two IRI patterns for different subsets.

Reply:
Thanks for the nice comments! I think for the purposes of this PR, we can call this done. But if we need to look into this again, this will be the first conversation to link to :)